data/human outfits.txt

```diff
@@ -3065,3 +3065,37 @@ outfit "City-Ship License"
 	category "Special"
 	thumbnail "outfit/city-ship license"
 	description "You look closely at the silhouette of the ship on the license, but you can't make out what model of ship it is. Perhaps it is an older city-ship, one of the first that carried humanity to greatness among the stars, or an intern just picked the first image that came up for 'city-ship'."
+	
+outfit "NDR-114 Android"
+	category "Systems"
+	licenses
+		Paragon

```

Comment:
i think im going to drop the license and instead just have this added to specific stations